requirements.txt

```diff
@@ -3,3 +3,4 @@ Django==2.0.4
 psycopg2==2.7.4
 pytz==2018.3
 django-heroku==0.3.1
+gdal==2.2.4
```

Comment:
oh interesting -- did this install everything on your end? The django guides for postGIS I saw said you had to install a bunch of stuff like GDAL manually, e.g. with `homebrew`

https://docs.djangoproject.com/en/2.0/ref/contrib/gis/install/#homebrew